tests/functional/test_scoping/rrel_multifile/navigation1.model

```diff
@@ -0,0 +1,18 @@
+#include "navigation1_included.model"
+A a1 {
+    A aa1 {
+        A aaa1 {}
+        A aab1 {}
+        P1 aaa1
+        P1 aa1
+        P1 ab1
+        P1 a2
+    }
+    A ab1 {}
+}
+A R {
+    A r2 {}
+}
+
+// activate importURI (in any case, also for `GrammarMissingPlusM.tx`):
```

Comment:
Why this comment here?

Reply:
I added an explanation. Look if it can be understood... We may to add more verbose explanation and a hint that "+m" will be deprecated...